.taskcluster.yml

```diff
@@ -1,25 +1,21 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+---
 version: 1
+reporting: checks-v1
 policy:
   pullRequests: public
 tasks:
   $let:
+    taskgraph:
+      branch: taskgraph
+      revision: f949fb56d96b9509b071ba55853f2d60e8dbbf57
```

Comment:
This looks good, using an up-to-date revision. I've been on a 4 months old for quite a while :)